daemon/daemon.go

```diff
@@ -980,6 +985,10 @@ func initNetworkController(config *Config) (libnetwork.NetworkController, error)
 	return controller, nil
 }
 
+func (daemon *Daemon) NetworkApiRouter() func(w http.ResponseWriter, req *http.Request) {
```

Comment:
If we don't want to import `het/http` in the daemon, regardless it's a good idea or not, we can export `daemon.netController` as public and make the call to `nwapi.NewHTTPHandler(daemon.NetController)` in the server. The bonus point is that we would not need this function either.